src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,783 @@
+    airportModels.add(new Fighter());
+    airportModels.add(new Bomber());
+    airportModels.add(new Helicopter());
+    airportModels.add(new Bushplane());
+
+    // Carrier gets to build all normal air units
+    UnitModel carrier = new SkyCarrier();
+    for (UnitModel um : airportModels)
+      carrier.baseActions.add(1, new UnitProduceLifecycle.UnitProduceFactory(um));
+
+    airportModels.add(new BigBoy());
+    airportModels.add(new GuncrossWing());
+    airportModels.add(new Kaputnik());
+    extras.add(new GuncrossRobot());
+    airportModels.add(carrier);
+
+    // Dump these lists into a hashmap for easy reference later.
+    kjwModels.shoppingList.put(TerrainType.FACTORY, factoryModels);
+    kjwModels.shoppingList.put(TerrainType.SEAPORT, kaijuModels);
+    kjwModels.shoppingList.put(TerrainType.AIRPORT, airportModels);
+
+    // Compile one master list of everything we can build.
+    for (UnitModel um : factoryModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : airportModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : kaijuModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : extras)
+      kjwModels.unitModels.add(um);
+
+    // Multiplies all movement (and ranges) by N
+    for( UnitModel um : kjwModels.unitModels )
+      um.baseMovePower *= KaijuWarsWeapons.KAIJU_SCALE_FACTOR;
+
+    return kjwModels;
+  }
+
+  public void registerStateTrackers(GameInstance gi)
+  {
+    super.registerStateTrackers(gi);
+
+    StateTracker.instance(gi, UnitTurnPositionTracker.class);
+
+    UnitResurrectionTracker rezzer = StateTracker.instance(gi, UnitResurrectionTracker.class);
+    // Populate resurrection pairs
+    GameReadyModels grms = gi.rules.unitModelScheme.getGameReadyModels();
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( null == umCast.resurrectsAs )
+        continue;
+
+      // We have a type that resurrects; find the type it rezzes to
+      for( UnitModel rezToModel : grms.unitModels )
+      {
+        if( !rezToModel.name.equals(umCast.resurrectsAs) )
+          continue;
+        rezzer.resurrectionTypeMap.put(um, rezToModel);
+        break;
+      }
+    }
+
+    SuicideAttackTracker kaputnikTracker = StateTracker.instance(gi, SuicideAttackTracker.class);
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( !umCast.suicideAttack )
+        continue;
+
+      kaputnikTracker.killTypeMap.add(um);
+    }
+  }
+
+  /*
+   * Units I do not plan to implement at this time
+
+   * Freezer (0-counter unit with 1 damage that slows targets by a tile; too complicated and bad)
+   * Food Cart (attracts kaiju within 3 tiles; not implementing kaiju path/targeting mechanics)
+   * Shark Jet (faster Fighter that moves after shooting; messy and either OP or UP)
+   * Cannon of Boom (immobile Rocket equivalent that teleports between labs; ew)
```

Comment:
Freezer and Cannon are both defined in the unit set. Is this comment outdated?

Reply:
Their weapons are defined, but not the units themselves.